legacy/core/src/main/java/com/fsck/k9/preferences/SettingsImporter.kt

```diff
@@ -100,6 +103,11 @@ class SettingsImporter internal constructor(
                 }
             }
 
+            if (preferences.getAccounts().size > 1 && !globalSettingsImported) {
+                K9.isShowUnifiedInbox = true
+                K9.saveSettingsAsync()
+            }
```

Comment:
`AccountCreator` and `SettingsImporter` now contain basically the same code. Since the code also directly references the `K9` object, I think it would be good to extract this code to a separate class.

```kotlin
/**
 * Configures the unified inbox after an account has been added.
 */
class UnifiedInboxConfigurator(
    private val accountManager: AccountManager,
) {
    fun configureUnifiedInbox() {
        if (accountManager.getAccounts().size > 1) {
            K9.isShowUnifiedInbox = true
            K9.saveSettingsAsync()
        }
    }
}
```

This new class can then be used by both `AccountCreator` and `SettingsImporter`.

Reply:
@cketti Done!